docs/modules/idrac_system_erase.rst

```diff
@@ -0,0 +1,188 @@
+.. _idrac_system_erase_module:
+
+
+idrac_system_erase -- Erase system and storage components of the server
+=======================================================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+Synopsis
+--------
+
+This module allows erasing System components such as iDRAC, BIOS, DIAG, etc., and Storage components such as PERC NV cache, Non-volatile memory, Cryptographic erase PD, etc. of the server.
```

Comment:
This module allows you to delete system components such as iDRAC, BIOS, DIAG, and so forth. You can also delete storage components such as PERC NV cache, non-volatile memory, cryptographic erase of physical disks, and so on of the server.

Reply:
Done

Reply:
This is generated by ansible-doc-extractor. Kindly ignore reviewing RST file moving forward
RST content is updated as per the module